securedrop_client/gui/auth/sign_in/button.css

```diff
@@ -0,0 +1,21 @@
+QWidget {
+    font-family: 'Source Sans Pro';
+}
+
+#SignInButton {
+    border: none;
+    background-color: #05edfe;
+    font-family: 'Montserrat';
+    font-weight: 600;
+    font-size: 14px;
+    color: #2a319d;
+}
+
+#SignInButton:pressed {
+    background-color: #85f6fe;
+}
+
+#SignInButton:disabled {
+    border: 2px solid rgba(42, 49, 157, 0.4);
+    color: rgba(42, 49, 157, 0.4);
+}
```

Comment:
This one comes from this other stylesheet (that we can't remove yet): https://github.com/freedomofpress/securedrop-client/blob/09cb3e01c59d343b1f31ce90c306a8dc67064109/securedrop_client/resources/css/modal_dialog_button.css#L17-L29